Library/ViewModels/PledgeViewModel.swift

```diff
@@ -59,22 +62,51 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
         (project, reward, isLoggedIn, reward.shipping.enabled, total)
       }
 
+    let userUpdatedReload = data.takeWhen(self.userSessionStartedSignal)
```

Comment:
Can I suggest this, just to keep the mapping of the `true`/`false` for reload in one place?

```swift
let userUpdatedData = data.takeWhen(self.userSessionStartedSignal)
  .map { data in
    (
      project: data.project,
      reward: data.reward,
      isLoggedIn: AppEnvironment.current.currentUser != nil,
      isShippingEnabled: data.isShippingEnabled,
      pledgeTotal: data.pledgeTotal
    )
  }

let initialLoad = data.take(first: 1).map { data in (data, true) }
let silentReload = data.skip(first: 1).map { data in (data, false) }
let userUpdatedReload = userUpdatedData.map { data in (data, true) }

self.pledgeViewDataAndReload = Signal.merge(
  initialLoad,
  silentReload,
  userUpdatedReload
)
```

Reply:
Did you want to still make this change?

Reply:
I already made it